### DISCUSSION TITLE
Website planning discussion issue

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/Face-X#1102] [General]
I am thinking of making a website of Face-X with all the features

**Comment by manisha-27** [comment]
Hey @akshitagupta15june , Can I work on this issue.
I'll be creating it using reactjs. Please assign me this issue.

**Comment by akshitagupta15june** [comment]
yes yes you can work on it
I am creating it as a discussion issue

**Comment by Anuran12** [comment]
@akshitagupta15june  Hi I want to make a website for this.
